services/bots/src/discord/commands/pong.ts

```diff
@@ -0,0 +1,13 @@
+import { TransformPipe } from '@discord-nestjs/common';
+import { Command, DiscordTransformedCommand, UsePipes } from '@discord-nestjs/core';
+
+@Command({
+  name: 'pong',
```

Comment:
Since all commands show up on the server, do we really need 3 test commands?

Reply:
/versions are now usefull, ping can be useful for a quick verification that the server is working properly.
Removed pong 👍